packages/terralib/tests/functional/basic/TerraLib.lua

```diff
@@ -24,17 +24,15 @@
 
 return {
 	TerraLib = function(unitTest)
```

Comment:
Checking if functions from source code were tested
Checking TerraLib.lua
Function 'TerraLib' was not tested.